CHANGES.md

```diff
@@ -7,7 +7,11 @@ project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
-_There are no unreleased changes at the moment._
+### Added
+
+* Add the ability for `--filename` to accept a directory. The client will upload the file
```

Comment:
Seems inconsistent with the other lines. 
Do we want changelog entries to be trailing with a point or not ?

Reply:
Is there an issue with this PR in particular? The majority of entries are like this. It might be worth normalizing the changelog as a whole, I'll see what I can do for the release.

Reply:
Nope, just point out inconsistencies.